src/assets/source/sass/style.scss

```diff
@@ -1,4 +1,7 @@
+@import url('https://fonts.googleapis.com/css?family=Lato:400,700');
```

Comment:
Add the stile in the file "public/index.html", you can see for example how Roboto font is added